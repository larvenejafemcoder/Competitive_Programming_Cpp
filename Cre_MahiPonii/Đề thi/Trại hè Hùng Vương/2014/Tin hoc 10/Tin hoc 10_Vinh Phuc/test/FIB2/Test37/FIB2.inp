100
2 1
19 602
22 18563
12 34
12 213
11 43
0 1
18 1396
34 756577
8 27
27 269595
20 4487
9 29
20 9935
28 294566
16 1452
19 3190
42 402120852
17 213
1 1
10 46
6 12
6 7
13 285
7 3
38 44944372
15 945
16 267
29 811414
27 181106
42 108067351
11 27
40 75447858
10 67
31 144014
6 2
22 20419
19 4147
45 896693021
29 109739
0 1
16 683
18 1580
34 5767498
22 11526
28 107296
38 62694084
3 1
35 13001036
5 8
14 178
36 2215477
1 1
24 32511
11 71
32 2865071
7 18
1 1
16 1193
0 1
10 28
27 266142
12 42
12 80
9 32
42 29122937
14 265
0 1
25 66079
37 34166609
43 378324345
19 850
39 91746607
2 1
42 371086974
26 14302
24 38432
30 1235472
17 2582
17 1232
38 35377038
42 173907127
41 52624487
45 1753935754
10 86
39 14277467
31 1097781
41 100150602
23 11307
22 16894
17 588
29 378904
43 358383075
43 323673783
11 99
43 345534581
8 29
44 609313579
16 1237
31 295863
