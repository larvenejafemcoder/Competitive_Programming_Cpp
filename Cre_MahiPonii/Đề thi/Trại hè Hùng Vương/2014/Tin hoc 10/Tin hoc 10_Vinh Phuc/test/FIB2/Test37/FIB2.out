1
230
7091
13
81
17
1
533
288987
10
102976
1714
11
3795
112514
555
1219
153596498
81
0
18
5
3
109
1
17167223
361
102
309933
69176
41278055
10
28818517
26
55009
1
7799
1584
342506257
41917
1
261
604
2202988
4403
40984
23947009
0
4965954
3
68
846237
0
12418
27
1094360
7
0
456
1
11
101657
16
31
12
11123972
101
1
25240
13050483
144507041
325
35044086
1
141742611
5463
14680
471908
986
471
13512826
66426612
20100766
669943844
33
5453507
419315
38254126
4319
6453
225
144729
136890154
123632384
38
131982466
11
232737077
473
113010
